js/Helper.elm

```diff
@@ -24,16 +24,19 @@ nextBid auction =
             auction.card.startingBid
 
 
-tryApplyCardEffect : Int -> Model -> Result String ( Model, Cmd Msg )
-tryApplyCardEffect index model =
+tryApplyCardEffectLocal :
```

Comment:
I factored this out so I can use it in `View.elm` to check if the card is activatable without having to supply the `toServer` function.